server/pom.xml

```diff
@@ -97,6 +97,7 @@
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-auth</artifactId>
+            <scope>${hadoop.scope}</scope>
```

Comment:
Is changing `hadoop-auth` scope only enough, what about others like `hadoop-common`, `hadoo-client`, `hadoop-yarn-xxx`?